### ISSUE TITLE
Frost DK raidframe

### ISSUE THREAD

**Issue by frankallen** [renstrom/NeavUI#79]
Hello,

Got an issue as a frost DK.
The raidframes are like all player are not in range (myself is not in range to myself?).

Does this anyone know yet?

**Comment by renstrom** [comment]
Hmm, am I right in assuming frost doesn't have Death Coil?

**Comment by frankallen** [comment]
Yes, Frost doesn't have Death Coil.

**Comment by renstrom** [comment]
Okay. Try removing this line and see if that helps:

https://github.com/renstrom/NeavUI/blob/master/Interface/AddOns/oUF_NeavRaid/modules/oUF_SpellRange/oUF_SpellRange.lua#L36

Alternatively change it like this:

```diff
-DEATHKNIGHT = { 47541 }, -- Death Coil (40yd) - Starter
+DEATHKNIGHT = { 61999, 47541 }, -- Death Coil (40yd) - Starter
```

Either solution requires reloading the UI.

**Comment by Grimsbain** [comment]
Should be fixed with #144.
